doc/lbaasv1-plugin-deploy-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: docs_page
+title: Using the F5 OpenStack LBaaSv1 Plugin in OpenStack
+url: {{ page.title | slugify }}
+resource: true
+tags: kilo, juno, icehouse, havana, lbaasv1, 1.0.10, 1.0.12 
+---
+
+## Overview
+
+The F5 OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP load balancing services -- including virtual IPs, pools, device service groups, and health monitoring -- in an OpenStack environment.
+
+The F5 plugin agent translates 'OpenStack' to 'BIG-IP', so to speak, allowing you to provision BIG-IP load balancing services in an OpenStack environment. 
+
+The diagram below shows a sample configured OpenStack environment using the F5 plugin for OpenStack LBaaSv1. The plugin agent communicates with a BIG-IP platform or Virtual Edition via iApps. The load balancing service request is handled by the BIG-IP according to its configurations; it can then connect, discover, and/or deploy to the cloud-based apps or vms in the OpenStack project network.
+
+<img src="/media/openstack_lbaas_env_example.png" alt="Figure 1" width="500">
+<strong>Figure 1</strong>
+
+### Prerequisites
+
+Before you install the F5 OpenStack LBaaSv1 plugin, you'll need the following:
+
+1. An [OpenStack](http://www.openstack.org/) **Kilo** installation that has at least one controller, compute, and network node. 
+**TIP:** If you're unsure where to begin with OpenStack, the [OpenStack Deployment Guide](tbd) is a good place to start.
+2. Licensed BIG-IP hardware or Virtual Edition (VE) (v11.5.0 or later). 
+3. Administrator priveleges on the OpenStack host(s), BIG-IP(s), and any associated device clusters. 
+
+## Deploy the F5 OpenStack LBaaSv1 Plugin
+
+### Install the Plugin Package
+
+The F5 OpenStack LBaaSv1 plugin is comprised of a driver and an agent. The driver should be installed on your controller. While the agent can run on any host that has the Neutron python libraries installed, we recommend installing it on a controller or gateway node (they contain the Neutron python libraries by default).   
+
+**TIP:** Install the agent on each host in your OpenStack environment; this will allow you to provision BIG-IP services across your network. Running multiple F5 plugin agents for the *same* environment on *different* hosts provides redundancy in LBaaS provisioning for that environment. 
+
+**CAUTION:** If you choose to deploy multiple agents for the same environment, they *must* run on different hosts. If you run multiple F5 plugin agents on the same host,  each agent must be orchestrating its own environment \(in other words, each agent has to be controlling different TMOS clusters with unique environment prefixes\).  
+
+#### Red Hat / CentOS
+```
+rpm -ivh f5-bigip-common_1.0.12.noarch.el7.rpm
+rpm -i f5-oslbaasv1-driver-1.0.12.noarch.rpm 
+rpm -i f5-oslbaasv1-agent-1.0.12.noarch.rpm
+```
+
+#### Ubuntu
+```
+dpkg -i f5-bigip-common_1.0.12_all.deb
+dpkg -i f5-oslbaasv1-driver_1.0.12_all.deb 
+dpkg -i f5-oslbaasv1-lbaas-agent_1.0.12_all.deb
```

Comment:
f5-oslbaasv1-lbaas-agent_1.0.12_all.deb should be f5-oslbaasv1-agent_1.0.12_all.deb